src/test/java/org/folio/service/profiles/MappingProfileServiceUnitTest.java

```diff
@@ -35,37 +38,40 @@
 @RunWith(VertxUnitRunner.class)
 @ExtendWith(MockitoExtension.class)
 @ExtendWith(VertxExtension.class)
-class MappingProfileServiceUnitTest {
+class MappingProfileServiceUnitTest extends RestVerticleTestBase {
```

Comment:
This is a unit test right, why does it need this super class?

Reply:
sure, will fix it